packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -177,7 +175,10 @@ export default class DCWebxdc {
         const ses = sessionFromAccountId(accountId)
         accounts_sessions.push(accountId)
         ses.protocol.handle('webxdc', (...args) =>
-          webxdcProtocolHandler(this.rpc, ...args)
+          webxdcProtocolHandler(this.rpc, appId, ...args)
+        )
+        ses.protocol.handle('maps', (...args) =>
+          mapProtocolHandler(this.rpc, appId, accountId, ...args)
```

Comment:
This reintroduces the bug with all apps using the same `appId`. See git blame.
Maybe I should have written it in a more future-proof way.